src/types/audit/auditQueryDataRequestDetails.ts

```diff
@@ -0,0 +1,16 @@
+export interface AuditQueryDataRequestDetails {
+  requesterEmail: string
+  requesterName: string
+  recipientEmail: string
+  recipientName: string
+  zendeskId: string
+  dateFrom: string
+  dateTo: string
+  identifierType: string
```

Comment:
Could we type this as an IdentifierTypes rather than string?

Reply:
No, because it could literally be anything: we're trying to cope with any requests, including invalid ones. We have to keep this very loose.